napalm_yang/parsers/base.py

```diff
@@ -123,9 +123,14 @@ def parse_list(self, attribute, mapping, bookmarks):
         for m in mapping:
             pdb = m.get("pdb", {})
             if pdb:
-                import pdb
-                pdb.set_trace()
-                continue
+                try:
```

Comment:
I don't mind this change but I'd rather do at the top of the file:

```
try:
    import ipdb
    pdb = ipdb
except ImportError:
    import pdb
```

and then just leave in here `pdb.set_trace()`

Reply:
either that or a helper function that does what you wrote. Probably the helper function is actually better to avoid loading this if it's not necessary. I mostly want to avoid duplication of code.